styleguide/template.md

```diff
@@ -113,6 +113,12 @@ To link to a header in a Markdown file in the same repo, use relative linking +
 
 - Example: [.NET Community](../docs/welcome.md#community)
 
+The C# language specification and the Visual Basic language specification are included in the .NET docs by including the source from the language repositories. The markdown sources are managed in the [csharplang](../csharplang) and [visual basic](../vblang) repositories.
+
+Links to the spec must point to the source directories where those specs are included. For C#, it's **~/_csharplang/spec** and for VB, it's **~/_vblang/spec**.
+
+- Example: [C# Query Expressions](~/_csharplang/spec/expressions.md#query-expressions)
```

Comment:
I see that it's like other examples in this document, but sadly, those examples do not help a lot. Those examples are rendered as links in the GitHub web UI. However, as the one who edits a document, I prefer to see *how* to produce a link.

Should we put examples in backticks: 
>- Example: `[C# Query Expressions](~/_csharplang/spec/expressions.md#query-expressions)`

OR combine both source text and produced link:
>- Example: `[C# Query Expressions](~/_csharplang/spec/expressions.md#query-expressions)` produces [C# Query Expressions](~/_csharplang/spec/expressions.md#query-expressions)

Reply:
See #8298